docs/undergrad.yaml

```diff
@@ -4,8 +4,8 @@
 # 1.
 Linear algebra:
   Fundamentals:
-    vector space: 'vector_space'
-    product of vector spaces: 'prod.semimodule'
+    vector space: 'module'
+    product of vector spaces: 'prod.module'
```

Comment:
Isn't that an unusual name? `module.prod` would be more standard.

Reply:
It is the `module` instance on the `prod` type, therefore `prod.module`.